content/guides/1.data-model/1.collections.md

```diff
@@ -110,6 +110,14 @@ Once configured, click :icon{name="material-symbols:sort" title="Sort Button"} i
 
 The **Save as Copy** option in :product-link{product="editor"} offers a way to effectively duplicate the current item. Duplication settings define which field values will be copied.
 
+::callout{icon="material-symbols:info-outline"}
+
+**Duplicating Items with Relationships**<br/>
+
+When you duplicate an item, its relationships are not copied. You must redefine these relationships for the new item.
```

Comment:
```suggestion
When you duplicate an item, any related items are not copied. You must create new relationships between the duplicated item and related items.
```

Reply:
"Relationships" is a hard word because it refers to the data modeling. But here it means related items.

Reply:
This is perfect, thank you!